ansible/pbTestScripts/vagrantPlaybookCheck.sh

```diff
@@ -105,6 +106,20 @@ checkVars()
                 echo "Can't find vagrant-rsync-back plugin, installing . . ."
                 vagrant plugin install vagrant-rsync-back
         fi
+	if [[ "$fastMode" == true ]]; then
+		skipFullSetup=",nvidia_cuda_toolkit"
+		case "$jdkToBuild" in
+			"jdk8" )
+				skipFullSetup="$skipFullSetup,MSVS_2017";
+				if [ "$buildHotspot" != "" ]; then
+					skipFullSetup="$skipFullSetup,MSVS_2010,VS2010_SP1"
+				fi
+				;;
+                	"jdk11" | "jdk13" | "jdk14" )
```

Comment:
Please use a default `*)` section here instead of listing all versions other than jdk

Reply:
Changed 👍